src/build.d

```diff
@@ -1173,7 +1173,7 @@ auto sourceFiles()
             parse.d parsetimevisitor.d permissivevisitor.d printast.d safe.d sapply.d scanelf.d scanmach.d
             scanmscoff.d scanomf.d semantic2.d semantic3.d sideeffect.d statement.d statement_rewrite_walker.d
             statementsem.d staticassert.d staticcond.d target.d templateparamsem.d traits.d
-            transitivevisitor.d typesem.d typinf.d utils.d visitor.d foreachvar.d
+            transitivevisitor.d typesem.d typinf.d utils.d visitor.d foreachvar.d chkprintf.d
```

Comment:
Isn't this list alphabetically sorted except one prior violation?

Reply:
I tend to just append new files, otherwise the entire sequence is disrupted and it's hard to see what changed. Alphabetizing them should be a separate PR, as that is a refactor.